airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json

```diff
@@ -38,7 +39,10 @@
             "properties": {
               "auth_type": {
                 "type": "string",
-                "const": "Client"
+                "const": "Client",
+                "enum": ["Client"],
```

Comment:
is this a UI bug? pretty sure artem made it so that this format is no longer needed in the UI

Reply:
yes, it's a UI bug. There is a video in the issue showing the problem :(

Reply:
can you create a follow up issue for artem if there isn't one?

Reply:
Hi. I believe there should be some info about this in the docs, somewhere at https://docs.airbyte.io/connector-development/connector-specification-reference maybe? Could you update it?